src/internal/interpreter/map-interpreter.errors.ts

```diff
@@ -99,6 +99,29 @@ export class HTTPError extends MapInterpreterErrorBase {
   ) {
     super('HTTPError', message, metadata);
   }
+
+  override toString(): string {
+    return [
+      `${this.kind}: ${this.message}`,
+      this.astPath ? `AST Path: ${this.astPath.join('.')}` : undefined,
+      this.metadata?.node?.location
+        ? `Original Map Location: Line ${this.metadata.node.location.line}, column ${this.metadata.node.location.column}`
+        : undefined,
+      this.request?.url ? `Request URL: ${this.request.url}` : undefined,
+      this.request?.headers
+        ? `Request headers:\n${Object.entries(this.request.headers)
```

Comment:
i wouldn't put this here, headers can contain sensitive information

Reply:
make sense 👍🏻

Reply:
What about response headers though?

Reply:
same. there's sensitive debug for this

Reply:
ok thanks, removed 🙌🏻

Reply:
Doesn't the url also contain sensitive info? Or is this the url before query parameters would be resolved?

Reply:
good point